android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -296,6 +297,7 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
             block()
         } catch (e: Exception) {
             Log.e(MODULE_NAME, "Error while executing command", e)
+            // TODO: re throw
```

Comment:
Please remove and create ticket for this

Reply:
Address in 554967836fc4e69ceaed9cca7bd9b446cee85ad8